system/runners/BDDRunner.cfc

```diff
@@ -153,7 +153,7 @@ component
 						e,
 						arguments.target,
 						arguments.testResults,
-						thisSuite
+						thisSuite?:""
```

Comment:
Can you change it to a ternary, there are still many issues with Adobe engines and elvis.

```
isNull( thisSuite ) ? {} : thisSuite
```